completion/pgfkeys.cwl

```diff
@@ -2,6 +2,7 @@
 # muzimuzhi, 7 Jul 2020, tikz-pgf 3.1.5b
 
 # from pgfkeys.code.tex
+\pgfkeysloaded#S
```

Comment:
It's an inner macro, hence probably not needed by any user.

Reply:
It's now been removed.